views/partials/search/npmPackages.html

```diff
@@ -0,0 +1,8 @@
+<ul class="search-results mx-0">
```

Comment:
as noted above, I think this should be named `npm-packages.html` for consistency.